go/systests/teams_test.go

```diff
@@ -1279,6 +1279,7 @@ func TestTeamCanUserPerform(t *testing.T) {
 	require.True(t, annPerms.DeleteChannel)
 	require.True(t, annPerms.RenameChannel)
 	require.True(t, annPerms.EditChannelDescription)
+	require.True(t, annPerms.ChangeChannelAvatar)
```

Comment:
It's not a channel avatar, that is not a thing that exists. The avatar is for the team. I think the root cause here is that `EditChannelDescription` is used in the teams code for teams related stuff, beyond this avatar problem (which is I guess why you were involved with it in the first place). You should make a new field here called `EditTeamDescription`, and use it for both the avatars and the actual team description.

You get the same error trying to edit the team description as a writer, but no black bar for some reason. cc @buoyad

Reply:
:white_check_mark: